src/_lib/modules/picker/picker.component.ts

```diff
@@ -378,15 +386,19 @@ export class NwPickerComponent implements OnInit, OnDestroy {
     }
 
     onFocus() {
-        this.showResults();
-        this.focus.emit(this.inputEl);
+        if (!this.isDisabled) {
+            this.showResults();
+            this.focus.emit(this.inputEl);
+        }
     }
```

Comment:
Can disabled elements be focussed?

Reply:
@garethdn if the disabled attribute is removed from the input element in the dom then they can focus and see results that's why i added the conditions to this function and the other one to prevent the focus and them from seen results.

Reply:
You mean if they manually remove the disabled attribute from the DOM element? Like via the inspector?

Reply:
Yes exactly, so we need to prevent that from happening even though not everyone is tech smart but i think its still good to guard against that unless you think differently ?

Reply:
I think it's overkill, but there's not much overhead in this case.

Generally speaking, I don't think it's a good idea to try to safeguard against users modifying DOM elements. If they know how to do this they might know how to modify requests before they're sent. They could do anything really. For users like this we'd rely on server-side safeguards; like validating that the user has the permission to do what they're trying to do.

Reply:
I agree with what you're saying but i think in some cases like in this one it is necessary to stop the user from accessing the options on the menu if they were to remove the disabled attribute. I think inspecting things in the developer tools is far more common for anyone to do than modifying a request 😄